command/operator_debug.go

```diff
@@ -251,9 +251,27 @@ func (c *OperatorDebugCommand) Run(args []string) int {
 		}
 	}
 
-	// Resolve server prefixes
-	for _, id := range argNodes(serverIDs) {
-		c.serverIDs = append(c.serverIDs, id)
+	// Resolve servers
+	members, err := client.Agent().Members()
+	c.writeJSON("version", "members.json", members, err)
```

Comment:
why "version" isn't this usually a path or dir?

Reply:
The first parameter to writeJSON is the directory name, so `members.json` is a file in a directory called `version` at the top level of the archive.

Reply:
Thanks - will take that into consideration when splitting into named targets.